flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -946,6 +948,19 @@ public void disconnectResourceManager(Exception cause) {
 	//  Internal methods
 	// ======================================================================
 
+	private CompletableFuture<TransientBlobKey> putTransientStream(InputStream inputStream, FileType fileType) {
```

Comment:
I would name this something like `putTransientBlobStream` to make clear that this is about Blob storage. Otherwise it sounds very non-specific.

Reply:
done.